app/assets/stylesheets/_buttons.scss

```diff
@@ -68,6 +67,22 @@ a .button,
 
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
+.download-button,
+.vocab-download-button {
+  float: left;
```

Comment:
These are just for the images right? Is there a reason they need floats to be positioned?

Reply:
@Dan @danieljauch without changing existing styles which include float properties the most accurate and flexible method seems to be retaining the float position property.

Reply:
@louise-r-blue did you take a look at the way the buttons on the search result cards are set up? They dont appear to use any floats and are basically the exact same thing at a smaller size.

Reply:
@Dan yes I my initial intention was to use it as a template but there is a lot more nesting happening in the search results page. I think the only way to emulate is to add more to the haml file?